doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -50,12 +50,12 @@ following Features tabs defined here.
 | Conformance                  | X             | Kubernetes conformance test |
 | None                         | X             | Kubernetes mainstream features |
 | Feature:DaemonSetUpdateSurge |               | Daemon set should surge pods onto nodes when spec was updated and update strategy is RollingUpdate |
```

Comment:
Link?

Reply:
I could not find any link for this. If you can find please add it to a comment.

Reply:
How about this one: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#rollingupdatedaemonset-v1-apps

Reply:
Sure, I can add this, however this is not a feature description but an API reference with a good description of the feature itself.